msfdb_ws

```diff
@@ -20,7 +20,7 @@ end
 def parse_args(args)
   opts = {}
   opt = OptionParser.new
-  banner = "msfdb - A remote database process for Metasploit Framework.\n"
+  banner = "msfdb_ws - Metasploit data store as a web service.\n"
```

Comment:
Is this name change just temporary until we do the omnibus msfdb integration?

Reply:
it can be